tests/CMakeLists.txt

```diff
@@ -257,7 +257,7 @@ FOREACH(_test ${_tests})
 	  | egrep -v '^--'
 	  | sed 's\#${ASPECT_BASE_DIR}\#ASPECT_DIR\#g'
 	  > ${CMAKE_CURRENT_BINARY_DIR}/output-${_test}/${_output}.notime
-	DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/output-${_test}/screen-output
+	DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/output-${_test}/${_output}
 		${CMAKE_CURRENT_BINARY_DIR}/output-${_test}/${_output}.cmp.notime
```

Comment:
Yeah, if you look at this command the only input is ${_output}$, so this should be the only dependency.